creator-node/src/services/stateMachineManager/stateReconciliation/index.js

```diff
@@ -9,7 +9,8 @@ const processJob = require('../processJob')
 const { logger: baseLogger, createChildLogger } = require('../../../logging')
 const handleSyncRequestJobProcessor = require('./issueSyncRequest.jobProcessor')
 const updateReplicaSetJobProcessor = require('./updateReplicaSet.jobProcessor')
-const recoverOrphanedDataJobProcessor = require('./recoverOrphanedData.jobProcessor')
+const recoverOrphanedDataJobProcessor =
+  require('./recoverOrphanedData.jobProcessor').default
```

Comment:
noob question - is this a best practice? i don't really see it widely used across our other ts modules